tests/e2e/performance/test_pvc_multi_clone_performance.py

```diff
@@ -0,0 +1,408 @@
+                log.info(f"Result yaml is {res}")
+                if res["status"]["phase"] == "Bound":
+                    log.info(f"{clone_name} Created and ready to use")
+                    break
+                else:
+                    log.info(
+                        f"{clone_name} is not ready yet, sleep 5 sec before re-check"
+                    )
+                    time.sleep(5)
+                    timeout -= 5
+            else:
+                raise Exception(f"Can not get clone status {res}")
+        if timeout <= 0:
+            raise Exception(
+                f"Clone {clone_name}  for {self.interface} interface was not created for 600 seconds"
+            )
+
+        create_time = self.measure_clone_creation_time(clone_name, start_time)
+
+        log.info(f"Creation time of clone {clone_name} is {create_time} secs.")
+
+        return create_time
+
+    def run_command(self, cmd):
+        """
+        Running command on the OS and return the STDOUT & STDERR outputs
+        in case of argument is not string or list, return error message
+
+        Args:
+            cmd (str/list): the command to execute
+
+        Returns:
+            list : all STDOUT / STDERR output as list of lines
+
+        """
+        if isinstance(cmd, str):
+            command = cmd.split()
+        elif isinstance(cmd, list):
+            command = cmd
+        else:
+            return self.params["ERRMSG"]
+
+        log.info(f"Going to run {cmd}")
+        cp = subprocess.run(
+            command,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            stdin=subprocess.PIPE,
+            timeout=600,
+        )
+        output = cp.stdout.decode()
+        err = cp.stderr.decode()
+        # exit code is not zero
+        if cp.returncode:
+            log.error(f"Command finished with non zero ({cp.returncode}) {err}")
+            output += f"{self.params['ERRMSG']} {err}"
+
+        output = output.split("\n")  # convert output to list
+        output.pop()  # remove last empty element from the list
+        return output
+
+    def run_oc_command(self, cmd, namespace):
+        """
+        Running an 'oc' command
+
+        Args:
+            cmd (str): the command to run
+            namespace (str): the namespace where to run the command
+
+        Returns:
+            list : the results of the command as list of lines
+
+        """
+
+        cluster_dir_kubeconfig = os.path.join(
+            config.ENV_DATA["cluster_path"], config.RUN.get("kubeconfig_location")
+        )
+        if os.getenv("KUBECONFIG"):
+            kubeconfig = os.getenv("KUBECONFIG")
+        elif os.path.exists(cluster_dir_kubeconfig):
+            kubeconfig = cluster_dir_kubeconfig
+        else:
+            kubeconfig = None
+
+        command = f"oc --kubeconfig {kubeconfig} -n {namespace} {cmd}"
+        return self.run_command(command)
+
+    def measure_clone_creation_time(self, clone_name, start_time):
```

Comment:
Good to have start_time!!!!! Its going to eliminate the unnecessary log collection, also expecting oc log command will exit faster with time.
We should enhance similar function in helper.py file with start_time param.

Reply:
Thanks for noticing this. One should definitely add start_time (and eventually other params) to corresponding helpers functions